pyannote/audio/embedding/models.py

```diff
@@ -172,3 +174,134 @@ def __call__(self, input_shape):
                             name="embedding_output")(x)
 
         return Model(input=inputs, output=embeddings)
+
+class TristouNette(object):
```

Comment:
Sorry for being a pain in the $*^%$ but I'd rather have two "unitary" pull requests: one dealing with models, one dealing with optimizers.